### PR TITLE
Add an academic integrity policy

### PR DESCRIPTION
**Pull request by tchebb**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -499,6 +499,52 @@ LO: Students should be able to...
 6. How does this project test itself?
 7. How does this project do CI, if at all?
 
+## Academic Integrity
+
+You are expected to adhere to [Tufts' Academic Integrity Policy][1] in this
```
Comment (tekknolagi) [inline index.md:504]: Consider using named footnote (`[...][^tufts-policy]`, I think)

File: index.md [hunk 2]
```diff
@@ -499,6 +499,52 @@ LO: Students should be able to...
...
+course. Plagarism of code or answers to assignments and sharing answers with or
```
Comment (tekknolagi) [inline index.md:505]: This sentence is a little clumsy

File: index.md [hunk 3]
```diff
@@ -499,6 +499,52 @@ LO: Students should be able to...
...
+accepting answers from others is strictly prohibited.
+
+You are allowed to reference Linux man pages and any open source projects while
+completing assignments. You are also allowed to read any internet resource,
+with the exception of material outside this website that explicitly pertains to
+this course (e.g. a former student's GitHub repo with assignment solutions).
```
Comment (tekknolagi) [inline index.md:511]: Consider also banning people from publishing such repos

**Review by tekknolagi — APPROVED** [review]
lgtm

File: index.md (hunk 3 above)
Comment (tchebb) [reply index.md:511]: Think this is covered in the opening paragraph: "... sharing answers with ...".

Comment (tekknolagi) [reply index.md:511]: Might be worth being explicit

**Pull request merged**